front-end/src/pages/Dashboard.js

```diff
@@ -1,16 +1,10 @@
 import React from 'react';
+import Project from "../components/Project";
 
 const Dashboard = () => {
   return (
-    <div
-      style={{
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-        height: '90vh'
-      }}
-    >
-      <h1>Dashboard</h1>
+    <div>
+      <Project />
```

Comment:
The dashboard just contains the project component but doesn't display any project information as state not set.